projects/openexr/project.yaml

```diff
@@ -0,0 +1,5 @@
+homepage: "https://openexr.com"
+language: c++
+primary_contact: "twodeecoda@gmail.com"
```

Comment:
do we have any confirmation / approval from the maintainers?

Reply:
Yes, I can forward you the email thread if you like.